requirements.txt

```diff
@@ -25,7 +25,7 @@ django-tables2==1.21.2
 django-tinymce4-lite==1.7.5
 django-two-factor-auth==1.9.1
 django-webpack-loader==0.6.0
-django_select2==7.1.1
+django_select2==7.2.0
```

Comment:
I see that 7.2.1 is the latest now.

Reply:
It is not yet released at pip https://pypi.org/project/django-select2/#history

Reply:
I have raised issue on the repo, most probably it will get fixed. https://github.com/applegrew/django-select2/issues/594

Reply:
@frjo 
The issue is resolved django-select2 released new version 7.2.2.
I have made changes in a branch to use `django-select2=7.2.2`